lib/CLI.rb

```diff
@@ -5,88 +5,98 @@
+		champs_by_role
+	  when 2
+		puts "Find a champion by name"
+		find_champ                    
+	  when 3
+		puts "List ALL champions by name"
+		show_all_champs
+	  when 4
+		puts "....Exiting! \n "
+		break
+	  end
+	end #end of while
+end #end of run
 
-        def show_all_champs
-            Champion.all.each { |champ|  puts champ.title }
-        end
-    
-         def find_champ
-            name = gets.strip.capitalize
-            temp = nil
-            temp = Champion.find_by_name(name)
-            if temp.abilities.empty?
-                getAbilities = getAbilities(temp.name) 
-                temp.abilities << getAbilities
-            end
-            binding.pry
-         end
+def show_all_champs
+  Champion.all.each { |champ|  puts champ.title }
+  end
+	
+def find_champ(name =nil)
+  if name == nil
+	 name = gets.strip
+  end
+	 temp = nil
+		   
+  if temp = Champion.find_by_name(name.gsub(/\w+/, &:capitalize))
+	if temp.abilities.empty?
+	  getAbilities = getAbilities(temp.name) 
+	  temp.abilities << getAbilities
+	end
+	  puts "     ______________________________________"
+	  puts "            #{temp.title}   "
+	  puts "     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"   		   
+	  puts temp.abilities.flatten.flatten
+	  puts "\n"
+			
+    else
+			 puts "   \n   Champion not found!   \n "
+  end
+end
 
-         def champs_by_role
-            get_roles = []
-            get_roles = Champion.list_all_roles
-                get_roles.each.with_index(1) { |x,y| puts "#{y}: #{x}"}
-                puts "Select which role"
-            input = gets.strip.to_i
-                #if input > 0 || input <= temp.size SHOULD ADD A CHECK 
-            role = temp[input-1] 
-            
-            get_roles = Champion.find_by_role(role)
-            get_roles.each_with_index(1) { |champ,index|
-                puts "#{index}: #{champ.name} " 
-                    if champ.abilities.empty?
-                        champ.abilities << getAbilities(champ.name)
-                    end
-            }
-              puts "would you like to select a champ from this list? yes/no"
-                choice = gets.strip
-                if choice == "yes"
-                    puts "Which number?"
-                    num = gets.strip.to_i
-                   puts Champion.find_by_name(temp[num-1].name).title
-                end
-                #binding.pry
-        end
+def champs_by_role
+  get_roles = []
+  get_roles = Champion.list_all_roles
+  get_roles.each.with_index(1) { |x,y| puts "#{y}: #{x}"}
+	puts "Select which role"
+	input = gets.strip.to_i
+	#if input > 0 || input <= temp.size SHOULD ADD A CHECK 
+	role = get_roles[input-1] 
+		 
+  get_roles = Champion.find_by_role(role)
+  get_roles.each.with_index(1) { |champ,index|
+	puts "#{index}: #{champ.name} " 
+  if champ.abilities.empty?
+    champ.abilities << getAbilities(champ.name)
+  end
+	}
+  puts " \n Would you like to select a champ from this list? yes/no \n"
```

Comment:
use of champ vs champion in other places in code

Reply:
im referring to champ on this line`puts " \n Would you like to select a champ from this list? yes/no \n"`